dask_ndmorph/_ops.py

```diff
@@ -1 +1,17 @@
 # -*- coding: utf-8 -*-
+
+
+import numpy
+
+import dask.array
+
+
+def _where(condition, x, y):
+    if isinstance(condition, (bool, numpy.bool8)):
+        dtype = numpy.promote_types(x.dtype, y.dtype)
+        if condition:
+            return x.astype(dtype)
+        else:
+            return y.astype(dtype)
```

Comment:
Note that `astype` is idempotent and has been for a long time (since Dask `0.8.2`). So this incurs no penalty.

xref: https://github.com/dask/dask/commit/7249133f5d0bbcd0d19c4a0cde25e2521c2ba982